pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aead_composite_decrypt.go

```diff
@@ -49,6 +49,11 @@ func (d *ECDHESAEADCompositeDecrypt) Decrypt(ciphertext, aad []byte) ([]byte, er
 		return nil, err
 	}
 
+	// TODO: add support for 25519 content encryption
```

Comment:
https://github.com/hyperledger/aries-framework-go/pull/1683/files#r414074468

Reply:
updated comment to reflect chacha content encryption